.golangci.yaml

```diff
@@ -0,0 +1,41 @@
+run:
+  timeout: 5m
+
+  build-tags:
```

Comment:
these seems largely knative-y

Reply:
That's b/c I just copied the strictest I was aware of from there 😉